socketio_middleware/src/rooms.rs

```diff
@@ -96,13 +100,24 @@ pub fn get_sockets_number_for_room(room_id: &str) -> usize {
 pub fn print_sockets_for_room(room_id: &str) {
     match ROOMS.get(room_id) {
         Some(sockets) => {
+            info!("ROOMS: room {} containt sockets number = {}.", room_id, sockets.len());
+            /*
             for socket in &*sockets {
                 debug!("ROOMS: room {} containted socketid {}, sockets number = {}.", room_id, socket.sid(), sockets.len());
             }
+            */
         }
 
         None => {
             debug!("ROOMS: no socket in room {}.", room_id);
         }
     }
 }
+
+///
+/// get rooms number
+///
+pub fn get_rooms_number() -> usize {
```

Comment:
Let's change this to `get_room_count` instead, just a bit clearer.

Reply:
OK, changed to get_room_count